### PR TITLE
WinUI RadioButtons

### PR DESCRIPTION
**Pull request by MartinZikmund**
GitHub Issue (If applicable): #

## PR Type

What kind of change does this PR introduce?

- Feature

## What is the current behavior?

Not supported.

## What is the new behavior?

Supported.

## PR Checklist

Please check if your PR fulfills the following requirements:

- [ ] Docs have been added/updated which fit [documentation template](https://github.com/unoplatform/uno/blob/master/doc/.feature-template.md) (for bug fixes / features)
- [ ] [Unit Tests and/or UI Tests](https://github.com/unoplatform/uno/blob/master/doc/articles/uno-development/working-with-the-samples-apps.md) for the changes have been added (for bug fixes / features) (if applicable)
- [x] Validated PR `Screenshots Compare Test Run` results.
- [x] Contains **NO** breaking changes
- [x] Associated with an issue (GitHub or internal) and uses the [automatic close keywords](https://help.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue).
- [x] Commits must be following the [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/#summary) specification.

### CODE REVIEW

**Review by carldebilly — APPROVED** [review]
Seems good for me. Nice PR.

File: src/Uno.UI/Microsoft/UI/Xaml/Controls/RadioButtons/RadioButtons.cs [hunk 1]
```diff
@@ -0,0 +1,740 @@
...
+namespace Microsoft.UI.Xaml.Controls
+{
+	[ContentProperty(Name = nameof(Items))]
+	public partial class RadioButtons : Control
+	{
+		public RadioButtons()
+		{
+			//__RP_Marker_ClassById(RuntimeProfiler.ProfId_RadioButtons);
+
+			var items = new List<object>();
```
Comment (dr1rrb) [inline src/Uno.UI/Microsoft/UI/Xaml/Controls/RadioButtons/RadioButtons.cs:24]:
```suggestion
			var items = new ObservableCollection<object>();
```

Comment (dr1rrb) [reply src/Uno.UI/Microsoft/UI/Xaml/Controls/RadioButtons/RadioButtons.cs:24]: I checked in the WinUI code, it's a `Vector`: `auto const items = winrt::make<Vector<winrt::IInspectable>>();` which are observable.

File: src/Uno.UI/Microsoft/UI/Xaml/Controls/Repeater/ViewportManagerWithPlatformFeatures.cs [hunk 2]
```diff
@@ -523,7 +523,9 @@ void EnsureScroller()
 
 		void UpdateViewport(Rect viewport)
 		{
-			global::System.Diagnostics.Debug.Assert(!m_managingViewportDisabled);
+			// Disabled for non-virtualizing layout in RadioButtons, may need to be revisited.
```
Comment (dr1rrb) [inline src/Uno.UI/Microsoft/UI/Xaml/Controls/Repeater/ViewportManagerWithPlatformFeatures.cs:526]: Is there an issue opened for that?

Comment (jeromelaban) [reply src/Uno.UI/Microsoft/UI/Xaml/Controls/Repeater/ViewportManagerWithPlatformFeatures.cs:526]: Nope will do.

**Review by dr1rrb — APPROVED** [review]

**Pull request merged**